### ISSUE TITLE
Make the dropdown and its data compatible with screen readers

### ISSUE THREAD

**Issue by egeakman** [rhventures/react-native-dropdown-selector#66]
The dropdown placeholder text, and its data (labels) should be read by screen readers.
